README.md

```diff
@@ -1,16 +1,181 @@
-# Title
+# Boolean Expressions
 
 ## Learning Goals
 
--SWBAT 1
--SWBAT 2
+* Use Equality Comparison
+* Use Inequality Comparison
+* Use Greater-Than Comparison `>`
+* Use Less-Than Comparison `<`
+* Use Greater-Than-or-Equal-To Comparison `>=`
+* Use Less-Than-or-Equal-To Comparison `<=`
+* Invert Truth Value with "Bang" (`!`)
+* Join Boolean Expressions with AND
+* Join Boolean Expressions with OR
 
 ## Introduction
 
-## SWBAT 1
+As we saw in the ternary expression, sometimes we need to get a Boolean value
+(`true` or `false`) _from_ an expression in order to use it _in_ another
+expression.  We showed that we can use  greater than (`>`) and less than (`<`)
+to perform comparisons that produce `true` or `false`. Let's learn more!
 
-## SWBAT 2
+## Arithmetic Comparisons
+
+### Use Equality Comparison
+
+To check whether two values are equal, we use the *equality operator* represented
+with `==` ("double-equal-sign"). If two values are equal, then the statement
+will return `true`. If they are not equal, then it will return `false`. For
+example:
+
+```ruby
+1 == 1 #=> true
+
+1 == 7 #=> false
+```
+
+**IMPORTANT**: The comparison operator* `==` *is distinct from the assignment
+operator* `=` *that is used to set a variable equal to a value. Mistaking these
+for each other is a common cause of unexpected behavior.
+
+### Use Inequality Comparison
+
+To check whether two values **are not** equal, we use the *inequality operator*
+represented with `!=` ("bang-equal-sign;" more on "bang" below). If two values
+are not equal, then the statement will return `true`. If they are equal, then
+it will return `false`. For example:
+
+```ruby
+1 != 1 #=> false
+
+1 != 7 #=> true
+```
+
+## Quantity Comparisons
+
+You might recall these from middle-school: comparisons of greater-than /
+greater-than-or-equal-to.
+
+### Use Greater-Than Comparison `>`
+
+If the value on the left of the operator is *greater than* the value on the
+right, then the evaluation is `true`; `false` otherwise.
+
+### Use Less-Than Comparison `<`
+
+If the value on the left of the operator is *less than* the value on the
+right, then the evaluation is `true`; `false` otherwise.
+
+### Use Greater-Than-or-Equal-To Comparison `>=`
+
+If the value on the left of the operator is *greater than or equal to* the
+value on the right, then the evaluation is `true`; `false` otherwise.
+
+### Use Less-Than-or-Equal-To Comparison `<=`
+
+If the value on the left of the operator is *less than or equal to* the
+value on the right, then the evaluation is `true`; `false` otherwise.
+
+## Invert Truth Value with "Bang" (`!`)
+
+The `!` operator inverts a truth value
+
+```ruby
+( 1 + 1 == 2) #=> true
+!( 1 + 1 == 2) #=> false
+```
+
+Since `1 + 1` evaluates to `2`; and since `2 == 2` the return value is `true`.
```

Comment:
I feel like single bang could use a second code snippet example to show that it isn't just for outside of parentheses. Or even a lead in to the double bang : `!true`